app/config/config.yml

```diff
@@ -136,4 +136,5 @@ twig:
         hypothesis_authority: '%hypothesis_authority%'
     paths:
         '%kernel.project_dir%/build': 'build'
+        '%kernel.project_dir%/vendor/elife/patterns/resources': 'patterns'
```

Comment:
Would rather use `ComposerLocator` rather than hard-coding the `vendor` path, but the expression language isn't available here and it's more work/less clear to work around it.